README.md

```diff
@@ -28,27 +45,29 @@ yarn install
 yarn chain
 ```
 
+This command starts a local Ethereum network using Hardhat. The network runs on your local machine and can be used for testing and development. You can customize the network configuration in `hardhat.config.js`.
+
 3. On a second terminal, deploy the test contract:
 
 ```
 yarn deploy
 ```
+This command deploys a test smart contract to the local network. The contract is located in `packages/hardhat/contracts` and can be modified to suit your needs. The `yarn deploy` command uses the deploy script located in `packages/hardhat/deploy` to deploy the contract to the network. You can also customize the deploy script.
 
 4. On a third terminal, start your NextJS app:
 
 ```
 yarn start
 ```
-
-Visit your app on: `http://localhost:3000`.
+Visit your app on: `http://localhost:3000`. You can interact with your smart contract using the contract component or the example ui in the frontend.
 
 Run smart contract test with `yarn hardhat:test`
 
 - Edit your smart contract `YourContract.sol` in `packages/hardhat/contracts`
 - Edit your frontend in `packages/nextjs/pages`
 - Edit your deployment scripts in `packages/hardhat/deploy`
 
-## Deploying Smart Contracts
+# Deploying your Smart Contracts
```

Comment:
"... to a live network" ?

Since we are already deploying locally in previous steps.